### DISCUSSION TITLE
Práctica HTML

### DISCUSSION THREAD

**Discussion by iguelAL** [Launch-X-Latam/MisionFrontEnd#217] [Q&A]
Hola, estuve viendo el live pasado y con base a eso comencé a hacer la práctica y quería saber su opinión y decirme que tal quedo la maquetación, gracias por sus opiniones jeje.

[link](https://github.com/iguelAL/Practica-HTML)

**Comment by EsthebanDiego** [comment]
Esta muy bien, igual solo por semantica yo cambiaria en  cliente.html la etiqueta article en donde estan los sabores la cambiara por una etiqueta div y en donde esta la direccion por una etiqueta address o una etiqueta footer.

**Reply by iguelAL** [reply]
Buena idea, muchas gracias por la opinión, lo voy a cambiar.
